### ISSUE TITLE
[NOTICE] New Documentation Repo

### ISSUE THREAD

**Issue by derrickmehaffy** [strapi/strapi#8898]
Hi everyone!

For a while now, we've been discussing how we could take Strapi's documentation to the next level. We know that the quality of the documentation has an important impact on your experience with Strapi.

So, we have gathered your feedback, worked on a detailed overview of the current documentation, listed what is missing and what should be improved, brainstormed to make the best decisions both for you and the documentation... Understand: **we are completely reworking the documentation!** 🎉

### What's to come?

You may have noticed that **we created a [new repository](https://github.com/strapi/documentation), `documentation`** . We decided not to keep the documentation with the code base, but instead to create a new repository from which the whole documentation will be managed.

The creation of this new repository implies:

- a migration of all the entire documentation
- a restructuring of the documentation - to make sure we have a proper base for the grand plans we have in mind
- the introduction of new processes - to make documentation contributions as clear and easy as possible for you

As we will soon start working on the items above, **we have made the decision to temporarily freeze all documentation contributions**. Contributions will be accepted again once the new repository is ready. We will keep you updated, hopefully we should be done for the beginning of 2021.

Don't hesitate to share your opinion on the Strapi [forum](https://forum.strapi.io/), we'll gladly discuss documentation with you. And thank you for being part of the Strapi community, and inspiring us to do better! 🚀

---

TLDR: Community documentation contribution is frozen for the end of 2020, we have a new documentation repo, and we will be moving to a new documentation URL structure. :wink:

**Comment by derrickmehaffy** [comment]
Forum thread for discussion: https://forum.strapi.io/t/notice-new-documentation-repo-and-2020-doc-change-freeze/1664

Locking Issue.

**Comment by derrickmehaffy** [comment]
Marking as closed as notice was up for 30 days
